site/en/blog/soft-navigations-experiment/index.md

```diff
@@ -0,0 +1,258 @@
+    </tr>
+    <tr>
+      <td>CLS</td>
+      <td>The largest window of shifts between the navigation times. As usual, this when the page is backgrounded as only then can the CLS be finalized. A 0 value is reported.</td>
+    </tr>
+    <tr>
+      <td>FID</td>
+      <td>FID is not currently reported by the <code>web-vitals</code> library.</td>
+    </tr>
+    <tr>
+      <td>INP</td>
+      <td>The INP between the navigation times. As usual this will be reported upon an interaction, or when the page is backgrounded as only then can the INP be finalized. A 0 value is not reported.</td>
+    </tr>
+  </tbody>
+</table>
+
+{% Aside 'warning' %}
+The `web-vitals` implementation is under active development and is subject to change with new changes being published to that branch. At present, it does not support FID, as that requires [more underlying work in Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=1407656) or polyfilling FID-based on events.
+
+Those using the `web-vitals` JavaScript library should be aware of this if looking to use this library on production sites, and should take a local copy and test that version, and monitor the branch for changes.
+{% endAside %}
+
+## Will these changes become part of the Core Web Vitals measurements?
+
+At present, this soft navigation experiment is exactly that—an experiment. We wish to evaluate the heuristics and see if they more accurately reflect the user experience before we make any decision on whether this will be integrated in the Core Web Vitals initiative. We are very excited at the possibility of this experiment, but cannot offer guarantees on whether or when this will replace the current measurements.
+
+We value web developers' feedback on the experiment, the heuristics used, and whether you feel it more accurately reflects the experience. The [soft navigation GitHub repository](https://github.com/WICG/soft-navigations/issues) is the best place to provide that feedback, though individual bugs with Chrome’s implementation of that should be raised in the [Chrome issue tracker](https://bugs.chromium.org/p/chromium/issues/list).
+
+### How will soft navigations be reported in CrUX?
+
+How exactly soft navigations will be reported in CrUX, should this experiment be successful, is also still to be determined. It is not necessarily a given that they will be treated the same as current “hard” navigations are treated.
+
+In some web pages, soft navigations are almost identical to full page loads as far as the user is concerned and the use of Single Page Application technology is just an implementation detail. In others, they may be more akin to a partial load of additional content.
+
+Therefore, we may wish to report these soft navigations separately in CrUX, or perhaps weight them when calculating the Core Web Vitals for a given page or group of pages. We may also be able to entirely exclude partial load soft navigation, as the heuristic evolves.
```

Comment:
I think we should end this line at "Therefore, we may wish to report these soft navigations separately in CrUX"

The rest is really speculation.

Reply:
or just remove it altogether, you already cover that its still to be determined and showcase why it may be different.

Reply:
There was a lot of debate about this wording so not keen to change again but let me think about it…